docs/content.zh/docs/ops/metrics.md

```diff
@@ -1581,6 +1581,11 @@ Note that the metrics are only available via reporters.
       <td>The number of failed materializations.</td>
       <td>Counter</td>
     </tr>
+    <tr>
```

Comment:
The "rowspan" should also be updated so that the format of the table could be showed correctly.

Reply:
Thanks @masteryhx , It's my mistake.